src/capture/frame.hpp

```diff
@@ -2,17 +2,19 @@
 
 #include <cstdint>
 #include <cstddef>
+#include <chrono>
 #include <memory>
 
 #include "size.hpp"
 
-
 namespace shar {
 
+using Clock = std::chrono::high_resolution_clock;
```

Comment:
I don't like that `#include "frame.hpp"` suddenly adds `Clock` in scope. I think we should make separate `time.hpp` file with time-related defines for: `Clock`, `TimePoint`, `Milliseconds`